src/main/resources/db/migration/0350/V387__ProjectMetricsUnit.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE accelerator.project_metrics
+    ADD COLUMN unit VARCHAR(25);
```

Comment:
Do we specifically want a database-enforced length limit here, as opposed to enforcing the limit in the application code?

IMO it'd be better to make this `TEXT` and enforce the length limit with a validation rule on the request payload. That way if a client passes in a value that's too long, it'll get back a validation failure error message rather than a generic HTTP 500 from a failed INSERT statement. See, for example, `CreateOrganizationRequestPayload`. One thing to know about using field validations is that you need a `@Valid` annotation on the request handler parameter (again, see the code for that endpoint for an example).

Reply:
Good thought, updated! I'm still trying to forget oracle world where everything is VARCHAR2 or CLOB, so I always think TEXT columns are CLOBs (which are stored very differently than everything else).